src/components/messenger/list/conversation-list-panel/conversation-list-panel.scss

```diff
@@ -58,7 +58,7 @@ $side-padding: 16px;
       }
 
       .messages-list__tab-badge {
-        display: none;
+        display: block;
```

Comment:
Is this change related to opening the last conversation bug?

Looks like this change is to fix a different issue, the unread count on favourites tab? If so, this should be fixed in a different pull request/commit. Try to keep commits separate

Reply:
ok, I will do this!